plugins/input-exercise/src/editor.tsx

```diff
@@ -95,32 +123,53 @@ export class InputExerciseEditor extends React.Component<
   }
   render() {
     const { readOnly, state, focused } = this.props
-    const { correctValue, wrongAnswers, type } = state
+    const { correctAnswers, wrongAnswers, type } = state
 
     return (
       <React.Fragment>
         {focused
-          ? renderIntoSidebar(
-              <React.Fragment>
-                <Dropdown
-                  label="Wähle den Antworttyp:"
-                  options={R.map(dataType => dataType.name, types)}
-                  onChange={this.handleTypeChange}
-                  value={this.translateDataType(type)}
-                />
-                <Input
-                  label="richtige Antwort:"
-                  value={correctValue}
-                  onChange={this.setCorrectValue}
-                />
-              </React.Fragment>
-            )
-          : null}
+				? renderIntoSidebar(
+				<React.Fragment>
+					<Dropdown
+						label="Wähle den Antworttyp:"
+						options={R.map(dataType => dataType.name, types)}
+						onChange={this.handleTypeChange}
+						value={this.translateDataType(type)}
+					/>
+				</React.Fragment>
+				) : null }
+				{readOnly ? (
+					<InputExerciseRenderer {...this.props} />
+				) : (
+					<React.Fragment>
+						<InputExerciseRenderer {...this.props} />
+						{correctAnswers.map((correctAnswer: CorrectAnswer, index: number) => {
+							return (
+								<div key ={index}>
+									<label>
+                 		richtige Antwort:
+										<input
+											type="text"
+                			value={correctAnswer.value}
+											placeholder="richtige Antwort eingeben"
+											onChange={this.correctAnswerChange(index)}
+										/>
+									</label>
+									<button onClick={this.removeCorrectAnswer(index)}>
+										Löschen {/* <FontAwesomeIcon icon={faTrashAlt} /> */}
+									</button>
+								</div>
+							)
+						})}
+           	<button onClick={this.addCorrectAnswer}>
+           		Richtige Antwort hinzufügen
+           	</button>
+          </React.Fragment>
+				)}
         {readOnly ? (
-          <InputExerciseRenderer {...this.props} />
+				null
```

Comment:
you can combine that with the previous `if`

Reply:
This change was made in the recent commit.